vispy/gloo/context.py

```diff
@@ -166,7 +166,12 @@ def flush_commands(self, event=None):
         """
         if self._do_CURRENT_command:
             self._do_CURRENT_command = False
-            self.shared.parser.parse([('CURRENT', 0)])
+            canvas = get_current_canvas()
+            if canvas and hasattr(canvas, '_backend'):
+                fbo = canvas._backend._vispy_get_fb_bind_location()
+            else:
+                fbo = 0
+            self.shared.parser.parse([('CURRENT', 0, fbo)])
```

Comment:
This looks good to me, but we need to update the GLIR spec in the wiki. I suspect the vispyjs implementation will just continue to work ok. cc @rossant